axelrod/strategies/memoryone.py

```diff
@@ -106,7 +84,7 @@ def __init__(self, p=None):
 
     def receive_tournament_attributes(self):
         (R, P, S, T) = self.tournament_attributes["game"].RPST()
-        if not self.p:
+        if self.p is None:
```

Comment:
what if `p` is 0? This would previously fail the condition, but is a valid input. Instead I check against `None`